packages/video_player_avplay/tizen/src/plus_player.cc

```diff
@@ -266,14 +266,10 @@ bool PlusPlayer::SetLooping(bool is_looping) {
 }
 
 bool PlusPlayer::SetVolume(double volume) {
-  LOG_INFO("[PlusPlayer] Volume: %f", volume);
-
-  if (GetState(player_) < plusplayer::State::kPlaying) {
-    LOG_ERROR("[PlusPlayer] Player is in invalid state");
-    return false;
-  }
```

Comment:
Is there a reason you removed the state check?

Reply:
I think we don't need state check, because if state is not right, it will return false when call setVolume.

Reply:
oh, I check the plusplayer source code, only dashplayer support set volume.
you can set vlume by below dash stream:
```dart
+        'https://dash.akamaized.net/dash264/TestCasesUHD/2b/11/MultiRate.mpd',
+        formatHint: VideoFormat.dash);
```

Reply:
> I think we don't need state check, because if state is not right, it will return false when call setVolume.

However, since we can't know the reason for setVolume failure and I think it is good to check the state according to the document because it looks consistent with other API implementations.

> oh, I check the plusplayer source code, only dashplayer support set volume.

Thank you for check. I checked with dash video.

Reply:
Ok, I have rolled back the state check code.
And I have implemented the code to set the volume of all streams.